src/App.js

```diff
@@ -26,7 +27,6 @@ const App = () => (
         </ul>
       </nav>
       <Routes>
-        <Route path="/" exact element={<Home />} />
         <Route path="/books" element={<Books />} />
```

Comment:
- [Optional] I would suggest you make your Books component a home/landing page. It is your app's landing page currently 👇 
![atilla redirect](https://user-images.githubusercontent.com/67087664/146540619-f11b2046-ff0e-42e7-b3df-1e718c7c08ec.png)